pkg/migrations/sql/idempotent/telemetry.sql

```diff
@@ -59,4 +59,73 @@ $$
         RETURN TRUE;
     END;
 $$
-LANGUAGE PLPGSQL;
\ No newline at end of file
+LANGUAGE PLPGSQL;
+
+CREATE OR REPLACE FUNCTION SCHEMA_PS_CATALOG.promscale_sql_telemetry() RETURNS VOID AS
+$$
+    DECLARE result TEXT;
+    BEGIN
+        SELECT (count(*) - 1)::TEXT INTO result FROM SCHEMA_PS_CATALOG.promscale_instance_information;
+        PERFORM SCHEMA_PS_CATALOG.apply_telemetry('instance_total', result);
+
+        SELECT count(*)::TEXT INTO result FROM SCHEMA_CATALOG.metric;
+        PERFORM SCHEMA_PS_CATALOG.apply_telemetry('metrics_total', result);
+
+        SELECT sum(hypertable_size(format('SCHEMA_DATA.%I', table_name)))::TEXT INTO result FROM SCHEMA_CATALOG.metric;
+        PERFORM SCHEMA_PS_CATALOG.apply_telemetry('metrics_bytes_total', result);
+
+        SELECT approximate_row_count('SCHEMA_CATALOG.series')::TEXT INTO result;
+        PERFORM SCHEMA_PS_CATALOG.apply_telemetry('series_total_approx', result);
+
+        SELECT n_distinct::TEXT INTO result FROM pg_stats WHERE schemaname='SCHEMA_TRACING' AND tablename='span' AND attname='trace_id';
+        PERFORM SCHEMA_PS_CATALOG.apply_telemetry('traces_total_approx', result);
+
+        SELECT approximate_row_count('SCHEMA_TRACING.span')::TEXT INTO result;
+        PERFORM SCHEMA_PS_CATALOG.apply_telemetry('spans_total_approx', result);
+
+        SELECT hypertable_size('SCHEMA_TRACING.span')::TEXT INTO result;
+        PERFORM SCHEMA_PS_CATALOG.apply_telemetry('spans_bytes_total', result);
+
+        SELECT count(*)::TEXT INTO result FROM SCHEMA_CATALOG.label WHERE key = '__tenant__';
+        PERFORM SCHEMA_PS_CATALOG.apply_telemetry('multi_tenancy_tenant_count', result);
+
+        SELECT count(*)::TEXT INTO result FROM timescaledb_information.data_nodes;
+        PERFORM SCHEMA_PS_CATALOG.apply_telemetry('node_count', result);
+
+        SELECT count(*)::TEXT INTO result FROM SCHEMA_CATALOG.label_key WHERE key = '__cluster__';
+        PERFORM SCHEMA_PS_CATALOG.apply_telemetry('ha_cluster_count', result);
+
+        SELECT count(*)::TEXT INTO result FROM SCHEMA_CATALOG.metric WHERE is_view IS true;
+        PERFORM SCHEMA_PS_CATALOG.apply_telemetry('downsampling_count', result);
+
+        SELECT count(*)::TEXT INTO result FROM SCHEMA_CATALOG.exemplar;
```

Comment:
shouldn't this be approximate?

Reply:
This should be <= number of metrics. Since num metrics is not approx, we shouldn't need here